### PR TITLE
Macca update #5

### PR DESCRIPTION
**Pull request by maccag69**
- **Added a FDG Encyclopaedia** *not done lots more to come and some unfinished subjects in there*
I feel like this is a good way to reference stuff and it provides a brief yet informative explanation to things related to FDG and FiveM 

- **Added a guide for potential fix to crashes**

- **Edited *How to Clip* and added guide on how to clip medal since it wasn't previously there**

- **Edited announcement renderer should look pretty** 

- **Added Graphic Modification md**

### CODE REVIEW

File: components/content/AnnouncementsRenderer.vue
Comment (Snakey42069) [inline components/content/AnnouncementsRenderer.vue:1]: This won't work. `image` does not exist on type `a`. Probably best to delete this change for now, however I do agree this page could do with some love. Check out https://api.fatduckgaming.com/announcements for an example of the data available.

**Comment by Snakey42069** [comment]
@maccag69 I'm debating if the Encyclopedia should be merged with the Dictionary. Thoughts on merging vs keeping separate?

**Comment by maccag69** [comment]
merging probaly i just didnt want to add to dictionary just incase it was a case off waffle

**Comment by maccag69** [comment]
@Snakey42069

**Comment by Snakey42069** [comment]
Copy. I'll review in the morning.

**Pull request merged**